README.md

```diff
@@ -45,7 +45,7 @@ Default: `false`
 
 Type: `bool`
 
-### `mssql_accept_microsoft_sql_server_2019_standard_eula`
+### `mssql_accept_microsoft_sql_server_standard_eula`
```

Comment:
@richm this PR changes the variable to remove `2019` from it. The role has been released to Galaxy already, should I keep the older `mssql_accept_microsoft_sql_server_2019_standard_eula` variable as an alias?

Reply:
Yes, good idea.

Reply:
Done in 7f0911d62e9eaf0e8016ccf009e6c0055b4133ac